docker-utils/startup.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -xeu
+
+cfgHome=/usr/local/zeebe/conf/zeebe.cfg.toml
+cfgLog=/usr/local/zeebe/conf/log4j2.xml
+
+if [[ "$DEPLOY_ON_KUBERNETES" == "true" ]]; then
+
+    DNSNAME="${HOST}.${DNS}"
+
+    sed -i "s/@DNSNAME@/${DNSNAME}/g" $cfgHome
+
+    if [[ -n $INITIAL_CONTACT_POINT && ! $HOST =~ .*-0$ ]]; then
+        sed -i "s/@INITIAL_CONTACT_POINT@/initialContactPoints = \[\n\t\"${INITIAL_CONTACT_POINT}\"\n\]/g" $cfgHome
+    else
+        sed -i "s/@INITIAL_CONTACT_POINT@//g" $cfgHome
+    fi
+
+else
+
+    sed -i "s/@DNSNAME@/$(hostname --ip-address)/g" $cfgHome
+    sed -i "s/@INITIAL_CONTACT_POINT@//g" $cfgHome
+
+fi
+
+if [[ "$DEBUG" == "true" ]]; then
```

Comment:
Just a thought.

In a long run I think this would be nice that it supports multiple logging levels and instead of using "boolean" here we could use a switch with LOGGING=[info,warning,debug,fatal].

Reply:
I second that.

Reply:
Before we build that I would like to investigate if we could environment variables in our log4j2 config file. I would prefer a solution without sed magic

Reply:
That sounds amazing! :+1:

Reply:
I think it is possible to use environment variables for that. I create issue for that https://github.com/zeebe-io/zeebe/issues/421.